cs/presenters.texy

```diff
@@ -545,10 +545,10 @@ Použití modelových tříd
 Jak jsme si řekli, model je samostatná vrstva a je složen z hromady tříd, kde každá obstarává část logiky aplikace. Pokud bychom měli například model `App\Articles`, který se stará o načítání a ukládání článků, mohli bychom si o něj v presenteru říct pomocí [Dependency Injection |dependency-injection], za předpokladu že je [registrovaný jako služba |configuring#sluzby] v [DI kontejneru |dependency-injection#nette-di-container].
 
 /--php
-class ArticlePresenter extends Presenter
+class ArticlePresenter extends Nette\Application\UI\Presenter
 {
-	/** @var \App\Articles @inject */
-	public $articles;
+	/** @var Articles @inject */
+	public $articleModel;
```

Comment:
Tady by to asi taky bylo vhodné nahradit za constructor injection (a upravit následující odsavec textu).